gluon/gluon-mesh-vpn-fastd/files/lib/gluon/announce/statistics.d/mesh_vpn

```diff
@@ -0,0 +1,28 @@
+local nixio = require('nixio')
+local ltn12 = require('luci.ltn12')
+local json = require('luci.json')
+local uci = require('luci.model.uci').cursor()
+
+local socket_path = uci:get('fastd', 'mesh_vpn', 'status_socket')
+local fastd_sock = nixio.socket('unix', 'stream')
+assert(fastd_sock:connect(socket_path))
```

Comment:
It would be nicer to just return `nil` when fastd is not running instead of causing an exception.